docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,176 @@
+# Commissioning nRF Connect Accessory using Android CHIPTool
+
+This article describes how to use [CHIPTool](../../src/android/CHIPTool/README.md) for Android smartphones to commission a Nordic Semiconductor nRF 52840 DK with installed
+[nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md) onto a CHIP-enabled Thread network.
+
+Prerequisites:
+- 2 x nRF 52840 DK
+- 1 x smartphone with Android 8+
+- 1 x PC with Ubuntu 20.04 and Docker installed (however, all the presented steps should be achievable in some way using other popular operating systems)
+
+The essence of the commissioning process is composed of the following steps:
+1. CHIPTool discovers the accessory device over BLE.
+2. CHIPTool establishes a secure connection to the device over BLE and sends Thread credentials.
+3. The accessory device joins a Thread network.
+
+Since a typical smartphone doesn't have native support for Thread networking, extra effort is needed to prepare the test environment. The article describes how to turn a PC with spare Wi-Fi card and nRF 52840 DK into a Thread border router to forward Wi-Fi packets to a Thread network. The below diagram depicts connectivity between CHIPTool and Lock Example Application:
```

Comment:
`Thread networking` -> `Thread radio`?

Reply:
Also add one sentence that CHIP accessory cannot be controlled over BLE, thus this section describes how IPv6 packets initiated from Mobile Phone can reach CHIP accessory over Thread.

Reply:
I have few comments on this part:
1. Diagram shows connectivity between devices running CHIPTool and Lock app not between applications at all. Moreover it actually shows whole network not only CHIPTool-Lock part. Maybe it would be better to write something like: "The below diagram shows connectivity between network components necessary to allow communication between devices running CHIPTool and Lock applications.
2. You didn't mention RCP before and then it appears on the diagram. I would write somewhere above that border router device is running RCP architecture application. It would be nice to expand RCP name once to make clear what is means.

Reply:
connectivity between network components... is a part of connectivity between the applications, so technically it's correct :) But if you think a user might misinterpret the diagram somehow I can extend the description according to your proposal.